### ISSUE TITLE
(Republication) Feature Request - 'Repair' and update manually function

### ISSUE THREAD

**Issue by KnifeCat** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#2674]
Hello there, I had an idea previously but they _shut me the door_ by @jostrander and I didn't really appreciate it.

## Feature Request - Repair function ##
Sorry if that might be a _'disrespect'_ sign or whatever stuff, but for more info of why I'm requesting this, #2628 will help you through. 

### Why? ###

The _(gosh)_ shortcut haven't created (for some reasons, and @jostrander has _disrespected me in some way or i don't know i don't usually talk english but still i'm very good at it my principal language is french_ and I have created @Alipoodle 's shortcut but, now if a mistake happen...y'know? Or if just something is broken, instead of uninstalling manually and reinstalling it, the repair would do it and keep G.P.M.D.P. saved data (like settings, account, etc.) in.

## Feature Request - Update Manually button or whatever stuff ##
I REALLY have a G.P.M.D.P. problem here! Once again, #2628 _MIGHT_ help you because on my non-Creators Update computer, no automatic updates. **WHA?!** And now don't answer me how to update it, i'm fine with it until the update button is there if it will be.

### Why? ###
**It's in the (and there if you read good) feature request function**

**Comment by MarshallOfSound** [comment]
@KnifeCat As far as I can see @jostrander did not disrespect you in any way in that linked issue.  In fact, in my opinion, the only person there being disrespectful is you.  In particular, the unneeded caps lock, self-censored swearing and quotes like this

> One day, the creator might check this.
> zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz

I also can see that you recently commented pointlessly on another issue #2570

I would like to bring your attention to the [CONTRIBUTING](https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/blob/master/CONTRIBUTING.md) document which outlines briefly how users should behave in the issue tracker.

I'm going to close this issue, if there is an actual with the shortcut not being created on the latest version (`4.4.0`) then please raise a concise and polite issue  **strictly** following the provided issue template.

**Comment by LeavingGitHub** [comment]
@MarshallOfSound That comment was an accident, and if you'd be _respectful_, you would forgive it. I do NOT know why, but it is there. I have deleted it, and **don't blame me now!** If it is an accident, it is an accident. _(simple or what? Who wouldn't understand that?)_ **But I have something to say.**


Most importantly, **WHAT DID I DO?! Like I said, I am NOT sure of if @jostrander DID disrespect me. I was not sure how to explain it, so I thought like this, but now _I shouldn't_.** And the worst is that **I put it in _ITALIC_**, and I HAVE MENTIONNED THAT I WAS **NOT SURE HOW TO EXPLAIN IT**. Simple or what?! Now what my reaction is **uninstalling G.P.M.D.P. and not using it. Anymore. Including DELETING EVERY COMMENTS I've made on this repository, and deleting my account.** This comment might be rude, but it is meant a bit to. I will now unsubscribe to this repository and say bye to GitHub. Now **I** am **not** the _disrespecting_ guy. _(i know it is **YOURS**'s opinion)_ I will now use MusicBee or use my other Google Play Music player, Player for GMusic from the Windows Store. Really, when I do this, it is something. I know I am not the giant here, but -1 user. The story of all this is that I started using it. I thought this would be nice _(and I got mistaken)_. I was all okay and then found the issue of #2628. I have posted it and on my other computer, it haven't got that issue. So I have commented another comment. Then @jostrander told me to uninstall it like any other program _(very respectful)_. So then I thanked him _(respect!)_. It was after that @Alipoodle have commented how to launch it manually. Then I thanked once again, then put another comment and a feature request. **There is the worst.** @jostrander have closed my issue and _**(i was not sure of it)**_ have left a command behind. Explicating this, here is the part of the description of this issue that explicate this.
_`i don't know i don't usually talk english but still i'm very good at it my principal language is french`_
So what now?! Need to say something? I will be unavailable until **tomorrow, the 31th July, 2017**.

Any last words, now before I leave??
